mlflow/openai/__init__.py

```diff
@@ -706,17 +705,17 @@ def get_params_list(self, data):
 
     def _construct_request_url(self, task_url, default_url):
         api_type = self.request_configs.get("api_type")
+        api_base = self.request_configs.get("api_base")
         if api_type in ("azure", "azure_ad", "azuread"):
-            api_base = self.request_configs.get("api_base")
             api_version = self.request_configs.get("api_version")
             deployment_id = self.request_configs.get("deployment_id")
 
             return (
                 f"{api_base}/openai/deployments/{deployment_id}/"
                 f"{task_url}?api-version={api_version}"
             )
-        else:
-            return default_url
+
+        return f"{api_base}/{task_url}" if api_base else default_url
```

Comment:
A small but important change. In the original code, `api_base` is only used for Azure.